src/components/generic/NavBar/components/NavBarLogout.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { IoLogOutOutline } from 'react-icons/io5';
+import { Button } from 'antd';
+
+interface NavBarLogoutProps {
+  onClick: () => void;
+  isLoading?: boolean;
+}
+
+const NavBarLogout: React.FC<NavBarLogoutProps> = ({ onClick, isLoading }) => {
```

Comment:
We decided to not use `React.FC` so you can just do this instead.
```suggestion
const NavBarLogout = ({ onClick, isLoading }: NavBarLogoutProps) => {
```